leapp/cli/report/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module is currently single workflow specific (upgrades). After introducing other use cases we can make it
+more robust
+"""
+
+from __future__ import print_function
+import os
+import sys
+
+from leapp.exceptions import CommandError
+from leapp.utils.clicmd import command, command_opt, command_arg
+from leapp.utils.report import fetch_upgrade_report_messages
+from leapp.cli.upgrade import fetch_last_upgrade_context
+
+
+@command('report', help='Create report for last upgrade execution (default) or specific execution id')
+@command_arg('id')
+@command_opt('--format')
```

Comment:
At least format, should be receiving some help string.

Reply:
Definitely. Thanks for pointing this out.